pyne/mesh.py

```diff
@@ -1076,12 +1083,12 @@ def common_ve_tags(self, other):
                                   self.structured_set,
                                   types.MBMAXTYPE,
                                   dim=3)
-        self_tags = self.mesh.tag_get_tags_on_entity(self_it.next())
+        self_tags = self.mesh.tag_get_tags_on_entity(self_it.__next__())
```

Comment:
Can we use `next(self_it)` like we did above for this? It would be considerably cleaner.

Reply:
Sure, will apply this suggestion in the next update.

Reply:
Sadly, using `next` to replace `__next__()` will cause error.

Reply:
This is because the `MeshSetIterator` isn't quite setup like a proper Python generator class. I've submitted a PR into your `python3_compatible` branch to fix that issue. Once that is merged, we can replace any other calls to `it.__next__()` with `next(it)`